node/setup.go

```diff
@@ -338,6 +338,10 @@ func createBlockchainReactor(
 	metrics *consensus.Metrics,
 ) (*p2p.ReactorShim, service.Service, error) {
 
+	if !cfg.BlockSync.Enable {
+		logger.Error("blocksync.enable = false, but Tendermint no longer allows blocksync to be disabled. This setting is now ignored and will be removed in the next version.")
```

Comment:
```suggestion
		logger.Error("blocksync.enable = false, but Tendermint no longer allows blocksync to be disabled. This setting is now ignored and will be removed in a future version.")
```

Reply:
Just in case we forget

Reply:
Technically we've already done this change so there's no chance we'll forget unless we revert the decision